Description: Fix bogus quotes in java paths when passed via environment
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: http://bugs.debian.org/774026
Bug-Debian: http://bugs.debian.org/774027
Last-Update: 2015-03-11

--- a/plugins/jvm/uwsgiplugin.py
+++ b/plugins/jvm/uwsgiplugin.py
@@ -44,12 +44,12 @@
            break
 
 try:
-    JVM_INCPATH = ['-I"' + os.environ['UWSGICONFIG_JVM_INCPATH'] + '"']
+    JVM_INCPATH = ['-I' + os.environ['UWSGICONFIG_JVM_INCPATH']]
 except:
     pass
 
 try:
-    JVM_LIBPATH = ['-L"' + os.environ['UWSGICONFIG_JVM_LIBPATH'] + '"']
+    JVM_LIBPATH = ['-L' + os.environ['UWSGICONFIG_JVM_LIBPATH']]
 except:
     pass
 
